kerastuner/engine/hyperparameters.py

```diff
@@ -195,7 +195,7 @@ class Int(HyperParameter):
     Args:
         name: Str. Name of parameter. Must be unique.
         min_value: Int. Lower limit of range (included).
-        max_value: Int. Upper limit of range (excluded).
+        max_value: Int. Upper limit of range (included).
```

Comment:
We should consider highlighting the fact that this is not the same behavior as `range`

Reply:
Done, added a note in the docstring:

```
Note that unlinke Python's `range` function, `max_value` is *included* in
the possible values this parameter can take on.
```